functions/strings.js

```diff
@@ -133,9 +133,9 @@ module.exports = {
          * Fixed set of suggestions
          */
         suggestions: [
-          'Jazz',
-          'Instrumental',
-          'Dance',
+          '<emphasis level="moderate">Jazz</emphasis>',
```

Comment:
I'd recommend to wrap `{{__resolvers.short-options.suggestions}}` with `<emphasis level="moderate">` instead. It would prevent duplication

Reply:
Good point. Made those changes, and rebased this branch onto master!